i18n/qqq.json

```diff
@@ -8,5 +8,10 @@
 	"github-desc": "{{desc|name=GitHub|url=https://www.mediawiki.org/wiki/Extension:GitHub}}",
 	"github-par-filename": "Label for the contents of the \"file name\" parameter\n{{Identical|Filename}}",
 	"github-par-branchname": "Label for the contents of the \"branch name\" parameter",
-	"github-par-reponame": "Label for the contents of the \"repository name\" parameter"
+	"github-par-reponame": "Label for the contents of the \"repository name\" parameter",
+	"github-par-lang": "Label for the contents of the \"lang\" parameter",
+	"github-par-line": "Label for the contents of the \"line\" parameter",
+	"github-par-start": "Label for the contents of the \"start\" parameter",
+	"github-par-highlight": "Label for the contents of the \"highlight\" parameter",
+	"github-par-inline": "Label for the contents of the \"inline\" parameter",
```

Comment:
Huh... now that I think about it I'm not sure if anything that uses this still exists. There used to be something like `#mapsdoc` on this page https://sandbox.semantic-mediawiki.org/wiki/Mapsdoc_-_leaflet but then for all declaratively defined parser hooks. I suspect this got ditched when migrating to https://github.com/JeroenDeDauw/ParserHooks and was never added again. Perhaps I'm forgetting about something though. @kghbln any idea?

So this is fine for now. However expanding the usage section of the README with descriptions of the new parameters would be good.